server/index.js

```diff
@@ -1,14 +1,24 @@
 const express = require('express');
+const db = require('../database/index.js');
+
 let app = express();
 
 app.use(express.static(__dirname + '/../client/dist'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-
+app.get('/api/locations', (req, res) => {
+  let productID = req.query.product_id;
```

Comment:
@gigicupp could this be done using fancy es6 destructuring?

Reply:
@masoncpott lol. next time. on the req.query, I am not really trying to extract anything else.